### ISSUE TITLE
Custom fields at attachments not working

### ISSUE THREAD

**Issue by almai** [aprokopenko/justcustomfields#69]
When I add new fields to attachments, JCF won't save the fields to the DB. I tried to add new fields or connect existing fields but all I can see on the attachments edit page is `<p>PR</p>`. When I edit the fields content and save, nothing happens and in the DB there's no new row added.

When I add new fields to posts, everything works well.

![medium_bearbeiten_ _dahoam_magazin_ _wordpress](https://cloud.githubusercontent.com/assets/402692/19645386/bc9f2094-99f4-11e6-87c6-744304fbe5f2.jpg)

**Comment by aprokopenko** [comment]
Attachment is not usual content type. We hide it from post types list in new version 3.0

**Comment by almai** [comment]
I wonder if it has any technical reason, because basically the only difference in the table wp_posts is one field (post_type) where it says "attachement" instead of "post".

**Comment by aprokopenko** [comment]
From DB structure there are no any limitations, however WordPress doesn't run a lot of hooks for media files, which executed for all other post types. So to make full custom fields integration need to do some things manually.
We never had an issue where we needed to add custom fields to attachments. And noone requested it before. So we just disabled this menu.
If we have time in future - we will check the support of attachments.
For now our next plan is Taxonomy custom fields and Address/Google maps component
